part-4/parameters-formulas-tags.md

```diff
@@ -0,0 +1,429 @@
+
+6. **In Children**: the condition is fulfilled by any model entity in all children of the parent container of the entity for which the criteria is defined. 
+
+    - For the parameter with the condition "In Children" located in `Organism|Container A`, the sum will include the following parameters:
+        - `Organism|Container A|Container A1|Param A`
+        - `Organism|Container A|Container A1|Volume`        
+        - `Organism|Container A|Container A1|Molecule A|Concentration`
+        - `Organism|Container A|Container A1|Molecule A|Param A`
+        - `Organism|Container A|Container A2|Param A`
+        - `Organism|Container A|Container A2|Volume`
+        - `Organism|Container A|Container A2|Molecule A|Concentration`
+        - `Organism|Container A|Container A2|Molecule A|Param A`
+     **AND** molecule amounts
+        - `Organism|Container A|Container A1|Molecule A`
+        - `Organism|Container A|Container A2|Molecule A`
+
+More than one condition can be combined for evaluation; the combinations are connected with a logical AND. The detailed procedures when and how to enter tag conditions are described in this chapter ([Sum Formulas](model-building-components.md#sum-formulas), [Transport Processes](model-building-components.md#transport-processes), [Observers](building-block-concepts.md#observers), [Events and Applications](model-building-components.md#events-and-applications)).
+
+Models generated in **PK-Sim**® make extensive **use of tags**: For example, open a PK-Sim® model and look under [Passive Transports](model-building-components.md#passive-transports) for one part of the blood flow through the organs of an organism called "MassTransferBloodPool2OrgPl". This is a passive transport process that occurs from the arterial plasma compartment to the plasma compartments of all organs except for the lung. Consequently, this transport process is occurring under the following conditions:
+
+1. Source container: tagged with "Arterial Blood" and tagged with "Plasma".
+2. Target container: tagged with "Plasma" and not tagged with "Arterial Blood" and not tagged with "Lung".
+
+Similarly, observers or events can be included or excluded from being created in different parts of the spatial structure. The molecule observer "Fraction excreted", for example, makes use of the tag "Urine", so this observer is only created in the urine container.
+
+## The Path concept
+
+Entities (parameters, molecule amounts, containers, reactions) in a model structure are accessed by their path. A path is a string that describes the location of an entity in the model structure hierarchy. The path elements are separated by the pipe character `|`. A path can be specified as either an absolute path or a relative path.
+
+- An **absolute path** specifies the complete path to a referenced object, e.g., `Organism| Organ|Volume`. An alias defined by an absolute path reference will always refer to the same object, regardless of where the formula is used.
```

Comment:
Remove blank in the example path. Also would be better to have a concrete organ name here, e.g. Organism|Bone|Volume